requirements-dev.txt

```diff
@@ -9,7 +9,11 @@ pytest
 pytest-cov
 pytest-env
 coverage
-pyld
```

Comment:
I guess this dependency is used in `tests/test_api.py`, the test is failing indeed https://github.com/geopython/pygeoapi/actions/runs/4396701156/jobs/7699332233

Reply:
Yes, I saw that it failed. Did not delete that dependency though. Or at least I wasn't aware that I did...
Anyway, will fix it of course 👍